pkg/api/types.go

```diff
@@ -262,15 +270,52 @@ type UnregisterAgent struct {
 	WasActive bool `json:"wasActive"`
 }
 
-// MoreResources is sent by the VM informant to the autoscaler-agent when the VM is in need of more
-// resources of a certain type
+// MoreResourcesRequest is the request type wrapping MoreResources that's sent by the VM informant
+// to the autoscaler-agent when the VM is in need of more resources of a certain type
+type MoreResourcesRequest struct {
+	MoreResources
+
+	// ExpectedID is the expected AgentID of the autoscaler-agent
+	ExpectedID uuid.UUID `json:"expectedID"`
+}
+
+// MoreResources holds the data associated with a MoreResourcesRequest
 type MoreResources struct {
 	// Cpu is true if the VM informant is requesting more CPU
 	Cpu bool `json:"cpu"`
 	// Memory is true if the VM informant is requesting more memory
 	Memory bool `json:"memory"`
 }
 
+// Not returns the field-wise logical "not" of m
+func (m MoreResources) Not() MoreResources {
```

Comment:
none of these methods are actually called, I think?

Reply:
Removed `Or`.